From f50a29cb6e4e83341e8036dda30ea0012ed459f4 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Tue, 9 Oct 2018 17:49:07 +0300
Subject: [PATCH 221/741] staging/fsl_qbman: remove bootmem header

commit 181ab9cdf3af43c96c9912874f88c5bfe867d811 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The bootmem allocator was removed in [1]. The memblock allocator is
supposed to be used directly instead. We already include it.

[1] afd505b ("mm: remove include/linux/bootmem.h")

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl_qbman/dpa_sys.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/staging/fsl_qbman/dpa_sys.h b/drivers/staging/fsl_qbman/dpa_sys.h
index e144f5a4f13d..66699426b2cd 100644
--- a/drivers/staging/fsl_qbman/dpa_sys.h
+++ b/drivers/staging/fsl_qbman/dpa_sys.h
@@ -36,7 +36,6 @@
 #include <linux/errno.h>
 #include <linux/io.h>
 #include <linux/dma-mapping.h>
-#include <linux/bootmem.h>
 #include <linux/slab.h>
 #include <linux/module.h>
 #include <linux/init.h>
-- 
2.17.1

